controls/3_1_control_plane_authn_and_authz.rb

```diff
@@ -0,0 +1,31 @@
+#
```

Comment:
Is the copyright header intended here?

Reply:
Nothing intended on my side, I was just working on files from "stupid" copy paste.
This header can be changed to suit what you find most appropriate.

Concerning the ids, I was just following what is now the section organisation in the Kubernetes CIS Benchmark 1.5 document (Control plane is a new section compared to 1.4).

I think the benchmark implementation should follow the document structure it refers to.
As you said, avoiding breaking changes can be managed by publishing new releases each time it is needed. So users using/looking for the X.Y version/implementation will always be able to get the benchmark implementation from a release while you can continue to follow the  benchmark evolution and publish new releases at any time.

Reply:
Agreed. Let's release a new version. Can you please remove the apache header? We use a common license file in the root directory and individual commits need to be signed off.

Reply:
The header has been removed from all the ruby files.